content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -363,6 +367,73 @@ will be terminated once the job backoff limit has been reached. This can make de
 from failed Jobs is not lost inadvertently.
 {{< /note >}}
 
+### Backoff limit per index {#backoff-limit-per-index}
+
+{{< feature-state for_k8s_version="v1.28" state="alpha" >}}
+
+{{< note >}}
+You can only configure the backoff limit per index for a Job, if you
+have the `JobBackoffLimitPerIndex` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+enabled in your cluster. A per-index backoff limit is only available for Indexed Jobs that
+use the `restartPolicy: Never`.
+{{< /note >}}
+
+When you run an [indexed](#completion-mode) Job, you can choose to handle retries
+for pod failures independently for each index. To do so, set the
+`.spec.backoffLimitPerIndex` to specify the the maximal number of pod failures
+per index.
+
+When the per-index backoff limit is exceeded for an index, Kuberentes considers the index as failed and adds it to the
+`.status.failedIndexes` field. The succeeded indexes, those with a successfully
+executed pods, are recorded in the `.status.completedIndexes` field, regardless of whether you set
+the `backoffLimitPerIndex` field.
+
+Note that a failing index does not interrupt execution of other indexes.
+You can additionally limit the maximal number of indexes marked failed by
+setting the `.spec.maxFailedIndexes` field.
+When the number of failed indexes exceeds the `maxFailedIndexes` field, the
+entire Job is marked failed by the Job controller, by setting the Failed
+condition in the Job status. This triggers termination of all running Pods for
+that Job.
+
+Once all indexes finish for a Job where you specified a backoff limit per index,
+if at least one of those indexes did fail, the Job controller marks the overall
+Job as failed, by setting the Failed condition in the status. The Job gets
+marked as failed even if some, potentially nearly all, of the indexes were
+processed successfully.
```

Comment:
I really like this paragraph. My only _suggestion_ would be to number the flow you are presenting:

```suggestion
Note that a failing index does not interrupt execution of other indexes.
You can additionally limit the maximal number of indexes marked failed by
setting the `.spec.maxFailedIndexes` field.

When the number of failed indexes exceeds the `maxFailedIndexes` field, the following happens:
1. The entire Job is marked failed by the Job controller, by setting the Failed
condition in the Job status. 
1. The termination of all running Pods for that Job is triggered.
1. Once all indexes finish for a Job where you specified a backoff limit per index,
if at least one of those indexes failed, the Job controller marks the overall
Job as failed, by setting the Failed condition in the status. 
1. The Job gets marked as failed even if some, potentially nearly all, of the indexes were
processed successfully.
```

Reply:
Rephrased to make it more clear flow, with a natural split into two scenarios, one per paragraph. The base line is when just `backoffLimitPerIndex` is specified. This is now the first paragraph. The second paragraph describes what happens when `maxFailedIndexes` is used and exceeded. wdyt?

Reply:
LGTM!

Reply:
Great, resolving.